libraries/utils/generics/applicationlogger.cpp

```diff
@@ -86,20 +87,29 @@ using namespace std;
     #define COLOR_FATAL         "\033[33;1m"        //yellow
     #define COLOR_RESET         "\033[0m"           //reset
 
-    #define LOG_WRITE(OUTPUT, COLOR, LEVEL, MSG) OUTPUT << COLOR << "" LEVEL " " << COLOR_RESET << MSG << "\n"
```

Comment:
@LorenzE Have you checked if there is still a white space or separation between keyword and actual massage?

Reply:
yes there is